inlong-audit/conf/application.properties

```diff
@@ -47,7 +47,7 @@ audit.kafka.consumer.name=inlong-audit-consumer
 audit.kafka.group.id=audit-consumer-group
 
 # Generic jdbc storage
-jdbc.driver=com.mysql.cj.jdbc.Driver
-jdbc.url=jdbc:mysql://127.0.0.1:3306/apache_inlong_audit?characterEncoding=utf8&useSSL=false&serverTimezone=GMT%2b8&rewriteBatchedStatements=true&allowMultiQueries=true&zeroDateTimeBehavior=CONVERT_TO_NULL
-jdbc.username=root
-jdbc.password=inlong
\ No newline at end of file
+audit.store.jdbc.driver=com.mysql.cj.jdbc.Driver
```

Comment:
please update https://github.com/apache/inlong/blob/master/inlong-audit/audit-docker/audit-docker.sh#L57-L58 at the same time.